usr/share/rear/skel/OBDR/etc/scripts/system-setup.d/43-init-cciss.sh

```diff
@@ -29,7 +29,7 @@ if [[ "$CDROM_DEVICE" && -b $CDROM_DEVICE ]]; then
 fi
 
 ### Find Host/Channel/Id/Lun of device
-HCIL="$(lsscsi | awk 'BEGIN {FS=""} / +cd\/dvd +HP +Ultrium/ { print $2, $4, $6, $8; exit }')"
+HCIL="$(lsscsi | awk 'BEGIN {FS="[\\]|\\[|:]"} / +cd\/dvd +HP +Ultrium/ { print $2, $3, $4, $5; exit }')"
```

Comment:
Could you add a comment what that command does
i.e. describe the implementation, cf.
"Code must be easy to read" and
"Code must be easy to understand"  in
https://github.com/rear/rear/wiki/Coding-Style

It is hard to imagine what goes on
when one does not have such hardware
so an original lsscsi line as a comment
would help a lot here.

Reply:
Thank you, I added comment with explanation, why we there is change in the code.